docs/ref/settings.txt

```diff
@@ -3029,6 +3029,19 @@ The list of validators that are used to check the strength of user's passwords.
 See :ref:`password-validation` for more details. By default, no validation is
 performed and all passwords are accepted.
 
+.. setting:: AUTH_USERNAME_VALIDATORS
+
+``AUTH_USERNAME_VALIDATORS``
+----------------------------
```

Comment:
I think this needs a short help text (like above for password validation) and then a .. versionadded:: 3.2